plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -37,7 +37,7 @@
 from ansible.module_utils.six.moves.urllib.error import HTTPError
 from ansible.module_utils._text import to_native
 
-URL_REALMS = "{url}/realms"
+URL_REALMS = "{url}/admin/realms"
```

Comment:
```suggestion
URL_REALMS = "{url}/auth/admin/realms"
```

Reply:
Yes you're right, this is the right URL to use but according to the example given at the line 504 of `keycloak.py` we should use https://auth.example.com/auth to call the module. So I think either the example must be change either URL_REALMS as I suggested. With your suggestion the URL used by the module will become  https://auth.example.com/auth/auth/admin/realms and this is not what we want. 
I hope my explanation was clear enough

Reply:
I think you mean `keycloak_realm.py` and not `keycloak.py`? In any case, in the examples of all keycloak modules, `/auth` is part of the URL.

Reply:
Yes sorry I meant `keycloak_realm.py` 
That's why I think using this line : `URL_REALMS = "{url}/admin/realms` would make the usage of the realm similar as the other